pyproject.toml

```diff
@@ -105,6 +105,7 @@ pyarrow = ["pyarrow"]
 pandas = ["pandas", "pyarrow"]
 duckdb = ["duckdb", "pyarrow"]
 ray = ["ray", "pyarrow", "pandas"]
+daft = ["getdaft>=0.2.12"]
```

Comment:
In Poetry you need to define `daft` as a requirement above, and you can reference it here. Does Daft ship with PyArrow by default?

Reply:
Daft does define a transitive dependency on pyarrow!